src/pages/ShoppingCart/ShoppingCart.scss

```diff
@@ -0,0 +1,96 @@
+@import '../../css/app.scss';
+
+.ShoppingCart {
```

Comment:
I have noticed that lately, you are using CamelCase for CSS rules, but I think the industry standard is to use kebab-case or something in the line of BEM CSS, is there a reason, or can we come back to kebab-case?

Reply:
Oh hmm, it's ok to continue using the standard kebab-case. Let's try how it looks 😁

Reply:
I have to change it in some files and we need to make another issue to refactor the rest of the files jeje since this is a long PR and will be hard to change it on all the files